### PR TITLE
[UXIT-1922] Ecosystem Explorer · Reorganise content folder [skip percy]

### PR DESCRIPTION
**Pull request by barbaraperic**
## Description
This PR simplifies the content structure for ecosystem explorer projects so they are now directly under the `ecosystem-explorer` directory, and not `ecosystem-explorer/projects`

## Key Changes
- Updated configuration in `config.yml` to reflect the new file paths
- Updated path configurations in `paths.ts`
- Updated content references in `home` and `grants` pages
- Added `.md` file filtering in `getMarkdownData` utility to ensure only markdown files are processed

## Testing
- [x] All project pages still load correctly
- [x] Project form submission works as expected
- [x] Featured projects on home and grants pages display properly
- [x] CMS can still access and manage project content

### CODE REVIEW

File: src/app/_constants/paths.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const PATHS = {
     {
       hasSubpaths: true,
       includesEntries: true,
-      customEntriesContentPath: '/ecosystem-explorer/projects/',
+      customEntriesContentPath: '/ecosystem-explorer',
```
Comment (mirhamasala) [inline src/app/_constants/paths.ts:101]: I believe we now don't need `customEntriesContentPath` anymore? Neither here nor anywhere else.

File: src/app/ecosystem-explorer/project-form/actions/submitProject.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import slugify from 'slugify'
 
-import { ECOSYSTEM_PROJECTS_DIRECTORY_PATH } from '@/constants/paths'
+import { ECOSYSTEM_EXPLORER_DIRECTORY_PATH } from '@/ecosystem-explorer/utils/getEcosystemProjectData'
```
Comment (mirhamasala) [inline src/app/ecosystem-explorer/project-form/actions/submitProject.ts:5]: Why did we change where we are importing this from?

**Review by mirhamasala — CHANGES_REQUESTED** [review]
Couple of comments! 🙏🏼

File: src/app/ecosystem-explorer/project-form/actions/submitProject.ts (hunk 2 above)
Comment (barbaraperic) [reply src/app/ecosystem-explorer/project-form/actions/submitProject.ts:5]: I followed the pattern we use in `getDigestArticleData` and `getEventsData` where we defined the DIRECTORY_PATH?

File: src/app/_constants/paths.ts (hunk 1 above)
Comment (barbaraperic) [reply src/app/_constants/paths.ts]: Correct! Removed it ✅

File: src/app/ecosystem-explorer/project-form/actions/submitProject.ts (hunk 2 above)
Comment (mirhamasala) [reply src/app/ecosystem-explorer/project-form/actions/submitProject.ts:5]: I meant this part `from '@/ecosystem-explorer/utils/getEcosystemProjectData'` - Why not keep `'@/constants/paths'`?

**Pull request merged**